Decision-Clothing Listener

francissmedley opened the pull request:
a listener for decision-clothing,

francissmedley committed:
Decision-Clothing Listener

a listener for decision-clothing

aliblackwell commented on projections/index.html:
```diff
@@ -86,12 +86,12 @@
 
 
-       <section class="view davidToLife">
+       <section class="view david-to=life">
```
equals sign should be a hyphen @francissmedley

aliblackwell commented on projections/projections-global-js/projection-router.js:
```diff
@@ -14,8 +14,8 @@ firebase.database().ref('view').on('value', function(snapshot) {
-		   case "/app/davidToLife/" :
-		   		showView('.davidToLife');
+		   case "/app/david-to-life/" :
+		   		showView('.david-to=life');
```
david-to-life!

francissmedley committed:
Remove Equals

fixed typos pointed out by ali

aliblackwell committed:
Resolve conflicts

# Conflicts:
#	projections/index.html
#	projections/projections-global-js/projection-router.js
